sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/Transforms.java

```diff
@@ -88,20 +105,43 @@ static TextDocumentBatchStatistics toBatchStatistics(RequestStatistics statistic
     }
 
     /**
-     * Helper method to set error details on {@link TextAnalyticsError}.
+     * Convert the incoming input {@link com.azure.ai.textanalytics.models.TextAnalyticsError}
+     * to a {@link TextAnalyticsException}.
+     *
+     * @param error the {@link com.azure.ai.textanalytics.models.TextAnalyticsError}.
+     * @return the {@link TextAnalyticsException} to be thrown.
+     */
+    static TextAnalyticsException toTextAnalyticsException(com.azure.ai.textanalytics.models.TextAnalyticsError error) {
+        String baseMessage = String.format(Locale.US, "%s: {%s}, %s",
+            "Status Code", HttpURLConnection.HTTP_OK, error.getMessage());
```

Comment:
Will this exception always be for 200 only?  A different exception is thrown for 4xx/5xx codes?

It's possible TA service could add another top-level error.  Does this exception type leave room to grow if they do?
Right now they have just these top-level errors:

          "enum": [
            "invalidRequest", // 400
            "invalidArgument", // 200
            "internalServerError", // 500
            "serviceUnavailable"  // 500
          ],

What exception type would we throw if they added another for 200?  Would you inherit from TextAnalyticsException or differentiate it another way?

Reply:
> Will this exception always be for 200 only? A different exception is thrown for 4xx/5xx codes?

Yes, we have `HttpResponseException` to deal with the unsuccessful response received with HTTP status code (e.g. 3XX, 4XX, 5XX) from the service request.

>What exception type would we throw if they added another for 200?

For another 200 added from the service side, we can update the `getMessage()` method to handle it accordingly. TextAnalyticsException can still be reused in those cases.

Also, on this, should we consider removing `Status code` from the exception message itself?